modules/setting-resource-quota-for-extended-resources.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * administering_clusters/setting-quotas-per-project.adoc
+
+[id='setting-resource-quota-for-extended-resources-{context}']
+= Setting resource quota for extended resources
+
+Overcommitment of resources is not allowed for extended resources, so you must
```

Comment:
@jeremyeder What is an "extended resource" exactly? I would like to include a brief definition here for clarity's sake. :thinking: